x/participation/keeper/msg_participate_test.go

```diff
@@ -37,14 +37,22 @@ func Test_msgServer_Participate(t *testing.T) {
 	// initialize auction
 	tk.Mint(sdkCtx, auctioneer, sdk.NewCoins(sellingCoin1))
 	auctionID1 := tk.CreateFixedPriceAuction(sdkCtx, r, auctioneer, sellingCoin1, startTime, endTime)
-	// initialize auction with edge case start time
+	// initialize auction with edge case start time (forcefully set to status standby)
```

Comment:
Just a feedback that I should have already given before. I would suggest using more explicit names depending on the test case for the variables. Using numbers makes the tests hard to read
Example
`auctionID1` -> `auctionRegistrationPeriod`
`auctionID3` -> `auctionStarted`